TFKUBE-135: Added the required dependency to fix the destroy order (#50)
atlassian-labs/data-center-terraform: docs/docs/userguide/CLEANUP.md

!!! warning "Make sure you have made backup of the product data."
    All product data will be **permanently deleted** after uninstall.

Please note the uninstall will destroy your data so make sure you have the latest backup of your data before you start cleaning the infrastructure.
You may proceed when you are ready to uninstall. 

```shell
./pkg/scripts/uninstall [-t] [-c <config.tfvars>]
```
Uninstall command removes the infrastructure but terraform state files will remain after the process. 

> You should use switch `-t` if you need to clean-up the terraform state as well.

Reviewer: ```suggestion
You should use switch `-t` if you need to cleanup the terraform state as well.
```

As you can manage multiple environments, you need to define the environment to uninstall. 
This is possible by using the same config file that you used to create the environment.  
If you have used the default configuration file (`config.auto.tfvars`) from the root folder of the project, then you may simply use the following command:

```shell 
./pkg/scripts/uninstall
```

> If you used a custom defined config file when the infrastructure is installed, then you need to run the following command using the same config file you used in install command instead:

Reviewer: ```suggestion
If you used a custom-defined config file when the infrastructure is installed, then you need to run the following command using the same config file you used in the install command instead:
```

```
./pkg/scripts/uninstall -c <custom-config-file>
```

> Uninstall by default will remove the products and all Atlassian Data Center infrastructure. 

Reviewer: ```suggestion
By default, uninstall will remove the products and all Atlassian Data Center infrastructure. 
```
